### ISSUE TITLE
Navigation exit goes to Login screen on front page books

### ISSUE THREAD

**Issue by mauno** [skodak/moodle-mod_book#18]
Hi Petr,

this is a very tiny issue and easy to fix. If you have a book on front page for guests and press at the last page of book nav_exit button moodle goes to Login screen if you are not logged in. This is caused by (line 159 in book 2.1 view.php)

```
$chnavigation .= '<a title="'.get_string('navexit', 'book').'" href="../../course/view.php?id='.$course->id.'#section-'.$sec.'"><img src="'.$OUTPUT->pix_url('nav_exit', 'mod_book').'" class="bigicon" alt="'.get_string('navexit', 'book').'" /></a>';
```

that could be changed (for example) to

```
if ($course->id = '1') {$chnavigation .= '<a title="'.get_string('navexit', 'book').'" href="'.$CFG->httpswwwroot.'"><img src="'.$OUTPUT->pix_url('nav_exit', 'mod_book').'" class="bigicon" alt="'.get_string('navexit', 'book').'" /></a>';$sec = '';}
 else {$chnavigation .= '<a title="'.get_string('navexit', 'book').'" href="../../course/view.php?id='.$course->id.'#section-'.$sec.'"><img src="'.$OUTPUT->pix_url('nav_exit', 'mod_book').'" class="bigicon" alt="'.get_string('navexit', 'book').'" /></a>';}
```

**Comment by skodak** [comment]
Hello, watch out for $course->id = '1' in your code, it would break on some older sites where frontpage course id is not 1.
Thanks for the report, I will patch it today.

**Comment by skodak** [comment]
fixed in 2.1 and master, thanks for the report!
